README.rst

```diff
@@ -4,8 +4,8 @@ coala-bears
 -----------
 
 coala-bears is a Python package containing all the bears that are officially
-supported by coala. It features more than **78 bears** covering
-**54 languages**. Here is a `generated list <https://github.com/coala/bear-docs/>`_
+supported by coala. It features more than **79 bears** covering
+**55 languages**. Here is a `generated list <https://github.com/coala/bear-docs/>`_
```

Comment:
https://github.com/coala/bear-docs/#supported-languages-64 according to the generated docs we have 64 languages. I think this might have to be updated seperately, the number of bears was never updated since a longer while